.gitmodules

```diff
@@ -44,15 +44,15 @@ fxDONOTUSEurl = https://github.com/ESCOMP/CISM-wrapper
 [submodule "rtm"]
 path = components/rtm
 url = https://github.com/ESCOMP/RTM
-fxtag = rtm1_0_80
+fxtag = rtm1_0_83
 fxrequired = ToplevelRequired
 # Standard Fork to compare to with "git fleximod test" to ensure personal forks aren't committed
 fxDONOTUSEurl = https://github.com/ESCOMP/RTM
 
 [submodule "mosart"]
 path = components/mosart
 url = https://github.com/ESCOMP/MOSART
-fxtag = mosart1.1.02
+fxtag = mosart1.1.05
```

Comment:
Mosart will be updated to mosart1.1.07

Reply:
@ekluzek I ended up submitting all the tests with mosart1.1.06 as I did in ctsm5.3.018. As before, the tests are passing because I entered the bug-fix manually to mosart and to rtm.

**This seems important:** I could update to mosart1.1.07 in this PR, but I would still need to enter the bug-fix manually to rtm.

So now I propose leaving this PR without a modules update. Then in #2052 I can update to mosart1.1.08 and rtm1_1_85, and I can put the bug-fix in that rtm tag. This way the bug-fix comes in for both mosart and rtm in a single ctsm tag.

What do you think? Let me know if you'd like to discuss it.

Reply:
That sounds like the right thing to do. The same change needs to go into CTSM, RTM and MOSART so you'll include that in the new tags for the ROF models you need to do anyway. So ctsm5.3.020 will have all of them updated for all the history changes as well as the rpointer stuff.